src/shortcuts/CreatorChild.jsx

```diff
@@ -95,7 +95,7 @@ export default class CreatorChild extends Shortcut {
                 // If the current attribute value contains this item in the ValueSet, don't include it in the options to select from
                 return !Lang.includes(attributeValue, item.name);
             }).map((item) => {
-                return {"key": item.id || item.code, "context": item.name, "object": item};
+                return {"key": item.id || item.code || item["MedDRA v12.0 Code"], "context": item.name, "object": item};

```

Comment:
Why do we have this hardcoded reference to MEDRA codes?

Reply:
In contextListOptions, on line 89 it passes context.key as the key for each context item. Almost all of the context have key except for adverse event because it uses "MedDRA v12.0 Code": 123245345 (some code) while the other context items use code: "1231231" (some code)

Reply:
I don't really understand what's happening there. If you have sometime today can you send me a message on Slack?

Reply:
Made a JIRA to deal with this later